modules/user-access/pages/sso.adoc

```diff
@@ -1,28 +1,35 @@
 = Single Sign-On
+:experimental:
+:sectnums:
 
 The Single Sign-On (SSO) feature in TigerGraph enables you to use your organization's identity provider (IDP) to authenticate users to access TigerGraph GraphStudio and Admin Portal UI.
 
-Currently we have verified following the identity providers which support SAML 2.0 protocol:
+We have verified the following identity providers that support SAML 2.0 protocol:
 
 * https://www.okta.com/[Okta]
 * https://auth0.com/[Auth0]
 
 For supporting additional IDPs, please inquire sales@tigergraph.com and submit a feature request.
 
-In order to use Single Sign-On , you need to perform four steps :
+In order to use Single Sign-On , you need to perform three steps :
 
 . Configure your identity provider to create a TigerGraph application.
-. Provide information from your identity provider to enable TigerGraph Single Sign-On .
+. Provide information from your identity provider to enable TigerGraph Single Sign-On.
 . Create user groups with proxy rules to authorize Single Sign-On users.
-. Change the password of the tigergraph user to be other than the default, if you haven't done so already.
```

Comment:
This is still needed, otherwise GraphStudio will login as tigergraph by default

Reply:
I listed this as a prerequisite instead. I think if a user is trying to do SSO, it should be expected that they should have enabled authentication.